console2/src/components/organisms/ProcessLogActivityV2/index.tsx

```diff
@@ -101,14 +107,46 @@ const ProcessLogActivityV2 = ({
         [location]
     );
 
+    const fetchProcessForm = useCallback(async () => {
+        const process = await apiGet(instanceId, ['checkpoints', 'history']);
```

Comment:
looks like only `process.status` needed.
maybe just use `processStatus` ?

Reply:
okay. and I don't need `process` here to check if the process is on? - https://github.com/walmartlabs/concord/pull/223/files#diff-a12794ac01d33384ead17750ed99ae1aee552896dffec038188897bf91e97274R136

Reply:
yep

Reply:
done!